sa/storage-authority.go

```diff
@@ -365,9 +370,16 @@ func (ssa *SQLStorageAuthority) countCertificatesByName(domain string, earliest,
 	} else if count > max {
```

Comment:
With this change that 10000 is also pretty magical now since we're likely to want folks to be able to go over it in the new rate limit we're going to build that's equivalent to the old one but with a higher default.

We'll have to rethink how we do our counting here to handle that higher rate limit.